src/org/labkey/test/tests/UserPermissionsTest.java

```diff
@@ -123,24 +123,44 @@ private void userPermissionRightsTest()
         _permissionsHelper.assertPermissionSetting(GAMMA_READER_GROUP_NAME, "No Permissions");
         _permissionsHelper.setPermissions(GAMMA_READER_GROUP_NAME, "Reader");
         createUserInProjectForGroup(GAMMA_READER_USER, PERM_PROJECT_NAME, GAMMA_READER_GROUP_NAME, false);
+
         //Create Author User
         clickProject(PERM_PROJECT_NAME);
         _permissionsHelper.enterPermissionsUI();
         _permissionsHelper.createPermissionsGroup(GAMMA_AUTHOR_GROUP_NAME);
         _permissionsHelper.assertPermissionSetting(GAMMA_AUTHOR_GROUP_NAME, "No Permissions");
         _permissionsHelper.setPermissions(GAMMA_AUTHOR_GROUP_NAME, "Author");
         createUserInProjectForGroup(GAMMA_AUTHOR_USER, PERM_PROJECT_NAME, GAMMA_AUTHOR_GROUP_NAME, false);
+
         //Create the Submitter User
         clickProject(PERM_PROJECT_NAME);
         _permissionsHelper.enterPermissionsUI();
         _permissionsHelper.createPermissionsGroup(GAMMA_SUBMITTER_GROUP_NAME);
         _permissionsHelper.assertPermissionSetting(GAMMA_SUBMITTER_GROUP_NAME, "No Permissions");
         _permissionsHelper.setPermissions(GAMMA_SUBMITTER_GROUP_NAME, "Submitter");
+
         // TODO: Add submitter to a group
         /*
          * I need a way to test submitter, I can't even view a folder where submitter has permissions when
          * impersonating on my local labkey, so may require special page?
          */
+    }
+
+    @Test
+    public void testSteps()
+    {
+        enableEmailRecorder();
+        userPermissionRightsTest();
+    }
```

Comment:
Just get rid of `testSteps` and make `userPermissionRightsTest ` the `@Test` method. Call `enableEmailRecorder ` from within it and update to be consistent with our test naming scheme (`testUserPermissionRights`)